bucket/filezilla.json

```diff
@@ -27,7 +32,7 @@
             "FileZilla"
         ]
     ],
-    "persist": "fzdefaults.xml",
+    "persist": "config",
```

Comment:
Can user change fzdrfaults.xml from UI?

Reply:
What will config directory contain?

Reply:
it contains the configuration files of filezilla (filezilla.xml, layout.xml, etc..). It's initialized by filezilla when no files exists at the first start.

Reply:
fzdefault.xml is not modifiable from the UI, the UI modify filezilla.xml file.